packages/product/src/services/product-category.ts

```diff
@@ -125,7 +124,7 @@ export default class ProductCategoryService<
   @InjectTransactionManager("productCategoryRepository_")
   async update(
     id: string,
-    data: ProductCategoryServiceTypes.UpdateProductCategoryDTO,
+    data: ProductTypes.UpdateProductCategoryDTO,
```

Comment:
nothing to do here for now, but just fyi you can remove most of this and rely on the new service base class. I think on the list and retrieve methods are extended to include the "descendant tree"

Reply:
Oh definitely, but I wouldn't dare touching this just yet, there are already too many changes 😄